### DISCUSSION TITLE
Vercel github bot showing deleted project

### DISCUSSION THREAD

**Discussion by salim-rca** [vercel/community#738] [Help] [answered]
Hi all,

I have 2 repos on GitHub and both were connected to separate Vercel projects.
I deleted 1 repo and the associated project on Vercel, but on the remaining repo the Vercel bot is still showing a check with the deleted project's name and a failed status on pull requests.

I tried giving the bot only access to the remaining repo instead of all repos in the GitHub settings but that didn't change anything. Is there any way to "refresh" the bot so it doesn't show the deleted project anymore?

Thank you

**Comment by amyegan** [comment]
Hi @salim-rca. I want to make sure I understand your problem. You're seeing a failed build status for the deleted project on __new__ PRs in a different project. Is that correct?

Can you share a screenshot of what you're seeing?

**Comment by salim-rca** [comment]
Hi @amyegan and thanks for replying.

You're correct in your understanding, here are screenshots:

- The automatic comment on new PRs from the bot is actually correct:
![image](https://user-images.githubusercontent.com/109099954/180049649-2916243e-5871-4dff-bb64-847d0e704d88.png)

- But the PR checks are not:
![image](https://user-images.githubusercontent.com/109099954/180049729-5ad90c61-a681-4978-9f1b-088eff1cfbc4.png)

"website" is the name of both the repo and Vercel project and they were both deleted. ("app" is the name of the current repo and Vercel project)

**Comment by amyegan** [comment]
Let's try the GitHub integration version of "turn it off and on again". You can disconnect and reconnect your GitHub repo from the Vercel Dashboard by going to your Project page -> Settings tab -> Git tab. Under "Connected Git Repository" click the "Disconnect" button. The section will update to show Git repo connection buttons. Click GitHub, search for your repo, and click the "Connect" button next to the one you want to reconnect to this Project.

**Comment by salim-rca** [comment, marked as answer]
Thanks for the suggestion! It didn't work at first, but then I tried to not only disconnect but also deleting the Vercel GitHub app from the repo. 

Everything looks good after a reconnect, thanks again @amyegan
